[2024-01-25T20:30:56.649+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cvd_automation_pipeline.read_vulnerability_file_task manual__2024-01-25T20:30:55.945469+00:00 [queued]>
[2024-01-25T20:30:56.657+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cvd_automation_pipeline.read_vulnerability_file_task manual__2024-01-25T20:30:55.945469+00:00 [queued]>
[2024-01-25T20:30:56.658+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-25T20:30:56.672+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): read_vulnerability_file_task> on 2024-01-25 20:30:55.945469+00:00
[2024-01-25T20:30:56.676+0000] {standard_task_runner.py:60} INFO - Started process 1429 to run task
[2024-01-25T20:30:56.679+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'cvd_automation_pipeline', 'read_vulnerability_file_task', 'manual__2024-01-25T20:30:55.945469+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/test_pipeline_dag.py', '--cfg-path', '/tmp/tmpq4xz2fju']
[2024-01-25T20:30:56.681+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask read_vulnerability_file_task
[2024-01-25T20:30:56.722+0000] {task_command.py:423} INFO - Running <TaskInstance: cvd_automation_pipeline.read_vulnerability_file_task manual__2024-01-25T20:30:55.945469+00:00 [running]> on host d1dec68e083c
[2024-01-25T20:30:56.792+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cvd_automation_pipeline' AIRFLOW_CTX_TASK_ID='read_vulnerability_file_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-25T20:30:55.945469+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-25T20:30:55.945469+00:00'
[2024-01-25T20:30:56.815+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-25T20:30:56.824+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=cvd_automation_pipeline, task_id=read_vulnerability_file_task, execution_date=20240125T203055, start_date=20240125T203056, end_date=20240125T203056
[2024-01-25T20:30:56.852+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-25T20:30:56.866+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
